docs/source/pages/DM1090.rst

```diff
@@ -14,6 +14,9 @@ The DM1090FFC baseboard has three FFC interfaces which allow for two 2-lane MIPI
 Please note that only revision R1M1E1 and newer camera modules can be connected to the DM1090FFC. With DM1090FFC we gained support for 22-pin RPi camera 
 interface. For that you will need a FFC from Arducam, which converts 26-pin Luxonis camera pinout to 22 pin RPi camera pinout.
```

Comment:
it would be nice to add a link to ArduCam FFC

Reply:
https://shop.luxonis.com/collections/all/products/rpi-hq-camera-imx477-adapter-kit <- Is this the one?